src/index.html

```diff
@@ -11,7 +11,106 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:3000/api" method="post">
+      <fieldset class="box">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label>Surname:
+            <input type="text">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>Name:
+            <input type="text">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>How old are You?
+            <input type="number"
+            min="1" max="100" value="12">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>Full date of birth:
+            <input type="date" placeholder="mm / dd / yyyy">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>I accept the term of the agreement
+            <input type="checkbox">
+          </label>
+        </div>
+      </fieldset>
+    </form>
+
+    <form>
+      <fieldset class="box">
+        <legend>Registration</legend>
+        <label>E-mail:
+          <input type="email" placeholder="email@example.com">
+        </label>
+        <br>
+        <label>Password:
+          <input type="password"
+          minlength="6" maxlength="30">
+        </label>
+      </fieldset>
+    </form>
+
+    <form>
+      <fieldset class="box">
+        <legend>An interesting fact about you!</legend>
+        <label class="label">Do you love cats?
+          <input type="radio"
+          name="animal" value="Yes">
```

Comment:
Если у тега больше двух атрибутов - размещай каждый на новой строке
```suggestion
          <input
            type="radio"
            name="animal"
            value="Yes"
          >
```